src/applications/personalization/dashboard-2/tests/e2e/claims-and-appeals.cypress.spec.js

```diff
@@ -29,14 +34,6 @@ describe('The My VA Dashboard Claims and Appeals section', () => {
         mockFeatureToggles();
         cy.visit(manifest.rootUrl);
 
-        // should show a loading indicator
```

Comment:
these just aren't needed. checking for the existence of the loading spinner is done in plenty of other tests